File: VB/DXBindingExample/ViewModel/RelativeSourcesViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations

Namespace DXBindingExample.ViewModel
    <POCOViewModel>
    Public Class RelativeSourcesViewModel
        Inherits BaseViewModel

        Public Overrides ReadOnly Property Title() As Object
            Get
                Return "Relative Sources"
            End Get
        End Property

        Public Overridable Property Value() As Integer
        Public Sub New()
            Value = 5
        End Sub
    End Class
End Namespace

File: CS/DXBindingExample/ViewModel/RelativeSourcesViewModel.cs
using DevExpress.Mvvm.DataAnnotations;

namespace DXBindingExample.ViewModel {
    [POCOViewModel]
    public class RelativeSourcesViewModel : BaseViewModel {
        public override object Title { get { return "Relative Sources"; } }

        public virtual int Value { get; set; }
        public RelativeSourcesViewModel() {
            Value = 5;
        }
    }
}
